src/components/Patient/PatientDetailsTab/Demography.tsx

```diff
@@ -364,11 +364,15 @@ export const Demography = (props: PatientProps) => {
                   size="medium"
                   variant="custom"
                   className={
-                    patientData.is_active
+                    !patientData.last_consultation?.discharge_date
```

Comment:
Keep the `is_active` check as well.

Reply:
@JavidSumra 

So the badge should be based on which facility data we are looking at, i.e.

Patient could be admitted to Dummy Facility 40, then discharged and admitted to Dummy Facility 2.

In this case, accessing patient's information from Dummy Facility 40 (under discharged patients list), should result in "Discharged", while accessing patient's information from Live patients or from Dummy Facility 2's list, should result in "Live".

So

```
const isPatientDischarged = () => {
    if (!patientData.is_active) return true;
    if (facilityId !== patientData.facility_object?.id.toString()) {
      return true;
    }
    return false;
  };
```